src/types.rs

```diff
@@ -77,6 +77,37 @@ pub struct CtlOperationAck {
     pub error: String,
 }
 
+/// A response containing the full list of known claims within the lattice
+#[derive(Clone, Debug, Default, Deserialize, Eq, PartialEq, Serialize)]
+pub struct GetClaimsResponse {
+    pub claims: Vec<HashMap<String, String>>,
+}
+
+/// A safety type that can handle claims values that are set to Null. This is
+/// only used internally for compatibility, we will still send out [GetClaimsResponse]
+/// as the response type.
+#[derive(Clone, Debug, Default, Deserialize, Eq, PartialEq, Serialize)]
+pub(crate) struct SafeClaimsResponse {
```

Comment:
I'm curious if we still need this? I don't think the new host passes null for any field: https://github.com/wasmCloud/wasmCloud/blob/main/crates/host/src/wasmbus/mod.rs#L3904

Reply:
We might not. Will give it a test with wadm and see what happens

Reply:
Yep, don't need it. It has been removed